src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -1483,7 +1502,7 @@ async function loadSilabsZcl(db, metafile, isJson = false) {
     if (ctx.version != null) {
       await zclLoader.recordVersion(db, ctx.packageId, ctx.version)
     }
-    await parseZclFiles(db, ctx.packageId, ctx.zclFiles)
+    await parseZclFiles(db, ctx.packageId, ctx.zclFiles, ctx)
```

Comment:
If we're passing the ctx as a final argument, then we should change the api to just get db and ctx, no need to have extra two separate arguments for ctx.packageId and ctx.zclFiles.

Minor nitpick. Not holding up the PR.....